torch/testing/_internal/common_methods_invocations.py

```diff
@@ -280,7 +280,7 @@ def supported_dtypes(self, device_type):
         if device_type == 'cpu':
             return self.dtypesIfCPU
         if device_type == 'cuda':
-            return self.dtypesIfROCM if TEST_WITH_ROCM else self.dtypesIfCUDA
+            return self.dtypesIfCUDA
```

Comment:
It looks like `dtypesIfROCM` is still set and used, so why does this code block not need to use it?

Reply:
Hi @jithunnair-amd ,
This basically sets that the dtypes will be equal to for that of CUDA.
So there is always parity between ROCM and CUDA.

Reply:
Hmm. This doesn't seem consistent with what this PR does (e.g. some `dtypesIfROCM` were still changed

Reply:
Yes, precisely, hence my comment above. Let me follow up offline with the PR author to get a bit more context.